Playbooks/playbook-Arcsight_-_Get_events_related_to_the_Case.yml

```diff
@@ -397,3 +462,7 @@ view: |-
     }
   }
 inputs: []
+outputs: []
+releaseNotes: updated deprecated tasks
+tests:
+  - No test - playbook
```

Comment:
why no test? 
please add test - this playbook is not deprecated and should be used